src/i18n/setup.js

```diff
@@ -0,0 +1,24 @@
+import i18n from 'i18next';
+import { initReactI18next } from 'react-i18next';
+import LanguageDetector from 'i18next-browser-languagedetector';
+import TranslationsEn from './translations/english.json';
+import TranslationsSp from './translations/spanish.json';
+
+const resources = {
+  en: { translation: TranslationsEn },
+  es: { translation: TranslationsSp },
+};
+
+i18n
+  .use(initReactI18next)
+  .use(LanguageDetector)
+  .init({
+    resources,
+    supportedLngs: ['en', 'es'],
+    fallbackLng: 'en',
+    detection: {
+      order: ['htmlTag', 'cookie', 'localStorage', 'path', 'subdomain'],
```

Comment:
order will change once we allow language to be changed.
I was thinking `['cookie', 'localStorage', 'path', 'subdomain', 'htmTag']`